### ISSUE TITLE
Torrent stalls while app window is open

### ISSUE THREAD

**Issue by lefty74** [astro/bitford#9]
I'm using Bitford on a Chromebook. I've noticed that the transfer rate dwindles to zero on some torrents. When I close the app window, Bitford keeps running as a background page (in the Chrome Task Manager) using about 500MB RAM and 100% CPU. When I reopen the app window, the torrent has made progress. The transfer rate starts off high, but again dwindles to zero over about 60 sec.
I'm attaching a screenshot showing a pretty long list of peers, but all at zero. The Pieces graph looks a bit odd to me, as they usually go straight from grey to green, only very briefly other colours. The graph you see was stuck like this: no movement.
Sorry if this is vague and non-technical: I'm not a coder. If you need more details please let me know.
![screenshot 2015-01-10 at 12 40 34](https://cloud.githubusercontent.com/assets/10475766/5691627/31d76936-98c9-11e4-92aa-ba00cc1bbe2d.png)

**Comment by astro** [comment]
Yes, back when I was developing this I noticed that a substantial amount of processing time goes into UI bookshopkeeping. Perhaps we're updating our model much too often for Angular.js. I don't know, very sorry. Perhaps it would help to limit to fewer peers.

**Comment by astro** [comment]
Saw that you signed up just for this bug report... I've got a ton of stuff on my hands and therefore cannot sink time in this project at the moment. Still, I've started contributing to webtorrent.io which is going to be the next generation of this stuff. It is going to be *that* much better, but give it time.

**Comment by lefty74** [comment]
Thanks for letting me know.

**Comment by lefty74** [comment]
I'm starring webtorrent. Different to this? http://jstorrent.com/

**Comment by astro** [comment]
Yes, entirely different project.

**Comment by lefty74** [comment]
Sorry, what I meant was, how are the projects different?
